Traceback (most recent call last):
  File "/Users/Yannick/opt/anaconda3/envs/ex/lib/python3.8/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/Users/Yannick/opt/anaconda3/envs/ex/lib/python3.8/site-packages/nbclient/client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/Users/Yannick/opt/anaconda3/envs/ex/lib/python3.8/site-packages/nbclient/util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/Users/Yannick/opt/anaconda3/envs/ex/lib/python3.8/site-packages/nbclient/util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "/Users/Yannick/opt/anaconda3/envs/ex/lib/python3.8/asyncio/base_events.py", line 616, in run_until_complete
    return future.result()
  File "/Users/Yannick/opt/anaconda3/envs/ex/lib/python3.8/site-packages/nbclient/client.py", line 663, in async_execute
    await self.async_execute_cell(
  File "/Users/Yannick/opt/anaconda3/envs/ex/lib/python3.8/site-packages/nbclient/client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/Users/Yannick/opt/anaconda3/envs/ex/lib/python3.8/site-packages/nbclient/client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
while True:
    
    x = int(input("Wie viele Arme hat ein Oktopus? Gib bitte eine Zahl an.\n"))

    if x < 8:
        print("Falsch, es sind mehr.")
    elif x == 8: 
        print("Ganz richtig!")
        break
    else:
        print("Nein, das sind zu viele.")
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mStdinNotImplementedError[0m                  Traceback (most recent call last)
[0;32m/var/folders/dy/1d_wflpx5_57vhd_6cdbfk0m0000gn/T/ipykernel_38086/2209010033.py[0m in [0;36m<module>[0;34m[0m
[1;32m      1[0m [0;32mwhile[0m [0;32mTrue[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m
[1;32m      2[0m [0;34m[0m[0m
[0;32m----> 3[0;31m     [0mx[0m [0;34m=[0m [0mint[0m[0;34m([0m[0minput[0m[0;34m([0m[0;34m"Wie viele Arme hat ein Oktopus? Gib bitte eine Zahl an.\n"[0m[0;34m)[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m      4[0m [0;34m[0m[0m
[1;32m      5[0m     [0;32mif[0m [0mx[0m [0;34m<[0m [0;36m8[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m

[0;32m/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/ipykernel/kernelbase.py[0m in [0;36mraw_input[0;34m(self, prompt)[0m
[1;32m   1001[0m         """
[1;32m   1002[0m         [0;32mif[0m [0;32mnot[0m [0mself[0m[0;34m.[0m[0m_allow_stdin[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m
[0;32m-> 1003[0;31m             raise StdinNotImplementedError(
[0m[1;32m   1004[0m                 [0;34m"raw_input was called, but this frontend does not support input requests."[0m[0;34m[0m[0;34m[0m[0m
[1;32m   1005[0m             )

[0;31mStdinNotImplementedError[0m: raw_input was called, but this frontend does not support input requests.
StdinNotImplementedError: raw_input was called, but this frontend does not support input requests.

